### PR TITLE
Format Go code in `pkg` and `test` too

### PR DESCRIPTION
**Pull request by pudelkoM**
Update the pull request CI job to format this newer code as well.

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -46,9 +46,8 @@ jobs:
           # https://github.com/mvdan/sh/issues/288
           shfmt -l -w $(shfmt -f . | grep -v vendor/)
       - name: Go Format
-        working-directory: pfcpiface
         run: |
-          go fmt
+          go fmt ./pfcpiface/... ./pkg/... ./test/...
```
Comment (osinstom) [inline .github/workflows/pull_request.yml:50]: shouldn't we use `make fmt` ?

**Review by osinstom — DISMISSED** [review]
Mergeable, once a comment is addressed.

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (pudelkoM) [reply .github/workflows/pull_request.yml]: yes, I also updated that one.

File: Makefile [hunk 2]
```diff
@@ -3,7 +3,6 @@
 
 PROJECT_NAME             := upf-epc
 VERSION                  ?= $(shell cat ./VERSION)
-GO_FILES                 := $(shell find . -type d \( -path ./pfcpiface/vendor -o -path ./pfcpiface/bess_pb  \) -prune -o -name '*.go' -print)
```
Comment (pudelkoM) [inline Makefile:6]: Vendoring is gone and `go fmt ./...` does the job of finding .go files on its own.

Comment (pudelkoM) [reply Makefile:6]: @osinstom I see you added this line, let me know if I missed something here

Comment (osinstom) [reply Makefile:6]: This will make `go fmt` format *.pb.go files too, but since they seem to be properly formatted, I'm fine with using `go fmt ./...`

**Review by osinstom — APPROVED** [review]

**Pull request merged**